src/data/polynomial/degree/basic.lean

```diff
@@ -770,18 +772,57 @@ lemma nat_degree_X_sub_C_le {r : R} : (X - C r).nat_degree ≤ 1 :=
 nat_degree_le_iff_degree_le.2 $ le_trans (degree_sub_le _ _) $ max_le degree_X_le $
 le_trans degree_C_le $ with_bot.coe_le_coe.2 zero_le_one
 
+lemma degree_sum_le_max {ι : Type*} {s : finset ι} (p : ι → polynomial R) :
+  degree (s.sum p) ≤ s.fold (⊔) ⊥ (λ i, degree (p i)) :=
```

Comment:
I'm a little surprised there's no way to use `⨆` or `Sup` here.

Reply:
It turns out this was the same as `degree_sum_le` anyway.